.editorconfig

```diff
@@ -4,3 +4,8 @@ end_of_line = LF
 indent_style = tab
 trim_trailing_whitespace = false
 insert_final_newline = true
+indent_size = 4
+
+[{*.json,*.yml,*.md}]
```

Comment:
oh, nice ... I didn't know this was possible.  We should apply this to everything cc @chasenlehara / @cherifGsoul